foundation/foundation-mda/src/main/resources/templates/general-docker/it.java.docker.file.vm

```diff
@@ -3,8 +3,7 @@
 # GENERATED DOCKERFILE - please ***DO*** modify.
 #
 # Generated from: ${templateName}
-ARG DOCKER_BASELINE_REPO_ID
-FROM ${DOCKER_BASELINE_REPO_ID}boozallen/openjdk:11-slim
+FROM openjdk:11-slim
```

Comment:
I: This image is not available in our ghcr.io registry, and the docker.io image is functionally what we need anyway. The 1.7 release notes address fixing this on upgrade.